src/ensembl/src/store.ts

```diff
@@ -35,18 +37,19 @@ const rootReducer = createRootReducer(history);
 export type RootState = StateType<typeof rootReducer>;
 
 const middleware = [
-  ...getDefaultMiddleware(),
   routerMiddleware(history),
   epicMiddleware,
-  analyticsMiddleWare
+  analyticsMiddleWare,
+  genomeBrowserApiSlice.middleware
```

Comment:
Should we considering [code splitting](https://redux-toolkit.js.org/rtk-query/usage/code-splitting) in this initial PR so that we can avoid adding individual middleware for each app?

Reply:
Code splitting is nice; but in the docs they are setting the `baseQuery` parameter, which defines both the root of the api endpoints and the protocol used to talk to these endpoints, in a single field in the empty api slice. We could probably hack around that, but it feels like a separate area of exploration.